### ISSUE TITLE
Improve CSS of all the input fields. 

### ISSUE THREAD

**Issue by plxity** [Awesome-React-Modules/React-Pincode#4]

**Comment by habeeba-naaz** [comment]
Hi...I am gssoc20 participant and interested to work on this issue...please assign me this issue if possible

**Comment by habeeba-naaz** [comment]
Please let me know when can I fork this project...?is test environment ready?

**Comment by plxity** [comment]
@habeeba-naaz  It will be done by tomorrow definitely

**Comment by habeeba-naaz** [comment]
Okay sure

**Comment by Rukmini-Meda** [comment]
I would like to work on this issue. Please assign this to me as part of GSSoC'20

**Comment by yellowwoods12** [comment]
Hi @habeeba-naaz, what is the progress on this issue? Let us know if you are working on it. Otherwise we might assign it to some other participant.

**Comment by habeeba-naaz** [comment]
Yes I am working...I got errors when I am linking to react-pincode...so I kept message in slack and waiting for reply

**Comment by plxity** [comment]
@habeeba-naaz Please follow the installation steps. You won't come across any errors.

**Comment by habeeba-naaz** [comment]
Now I don’t have errors...I am working on it

**Comment by Kuljeet-123** [comment]
@plxity  I would like to work on this issue ...can you please assign me this.

**Comment by Rukmini-Meda** [comment]
@plxity @arpit9667 @AdityaSrivast Can I work on this issue? I have commented on this earlier as well. Please assign as part of GSSoC'20

**Comment by Rukmini-Meda** [comment]
Hello mentors, I need some help. Do we have to run npm run build in project and test folders every time to reflect the changes we make in the project folder?

**Comment by AdityaSrivast** [comment]
@Rukmini-Meda you'll have to run `npm run build` in the parent folder after making changes in the module. No need to stop the react server in test directory while doing it. The changes should be reflected in the test server.

**Comment by Rukmini-Meda** [comment]
Oh okay. Thank you!
